roles/splunk_search_head/tasks/main.yml

```diff
@@ -50,6 +50,14 @@
   vars:
     app_list: "{{ splunk.apps_location }}"
 
+- include_tasks: ../../../roles/splunk_common/tasks/provision_apps.yml
+  when:
+    - "'app_paths_install' in splunk and splunk.app_paths_install.default"
```

Comment:
Will there ever be a case where `default` is not a key in `splunk.app_paths_install`? Should we also check for `default in splunk.app_paths_install`?

Reply:
'getSplunkAppPathInstall()' fills out 'app_paths_install' for all types so it will exist if 'app_paths_install' exists.